manifests/zone.pp

```diff
@@ -14,15 +14,16 @@
 #  *$zone_ns*: Valid NS for this zone (master only)
 #  *$zone_xfers*: IPs. Valid xfers for zone (master only)
 #  *$zone_masters*: IPs. Valid master for this zone (slave only)
+#  *$zone_forwarders*: IPs. Valid forwarders for this zone (forward only)
 #  *$zone_origin*: The origin of the zone
 #  *$zone_notify*: IPs to use for also-notify entry
 #
 define bind::zone (
   $ensure          = present,
   $is_dynamic      = false,
-  $is_slave        = false,
```

Comment:
@mcanevet @raphink proposal: keep the $is_slave for backward compatibility, and use it in order to set an internal value (with a mix of new $zone_type) used in the "case" bellow.
Code won't be much heavier, and this would prevent crashes on our infra (and for other users).
Is this a good idea?